src/MatBlazor/Components/MatIconButton/BaseMatIconButton.cs

```diff
@@ -48,6 +48,13 @@ public bool Toggled
         [Parameter]
         public EventCallback<bool> ToggledChanged { get; set; }
 
+        /// <summary>
+        /// The default behavior of the button. Possible values are:
+        /// "button", "reset", "submit"
+        /// </summary>
+        [Parameter]
+        public string Type { get; set; }
```

Comment:
Might be good to set the default to "button" ??

`public string Type { get; set; } = "button";`